Content.Server/Procedural/DungeonJob.Generator.cs

```diff
@@ -184,8 +184,8 @@ private async Task<Dungeon> GeneratePrefabDungeon(PrefabDunGen prefab, EntityUid
                         {
                             for (var y = roomSize.Bottom; y < roomSize.Top; y++)
                             {
-                                var index = matty.Transform(new Vector2(x, y) + grid.TileSizeHalfVector - packCenter).Floored();
-                                tiles.Add((index, new Tile(_tileDefManager["FloorPlanetGrass"].TileId)));
+                                //    var index = matty.Transform(new Vector2(x, y) + grid.TileSizeHalfVector - packCenter).Floored();
+                                //    tiles.Add((index, new Tile(_tileDefManager["FloorPlanetGrass"].TileId))); // This creates grass in space and places it should not have grass in it
```

Comment:
how does this affect normal planet dungeon generation?

Reply:
Not at all since this is only on space ones and also dungeons only have grass on grass planets, that spawn as part of the floor.